src/aave-v2/SupplyVaultBase.sol

```diff
@@ -39,33 +35,40 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
     /// @notice Thrown when the zero address is passed as input or is the recipient address when calling `transferRewards`.
     error ZeroAddress();
 
-    /// CONSTANTS AND IMMUTABLES ///
```

Comment:
I think that it was to have the same comment everywhere (we could put constant only btw, this is the same thing).

Reply:
It's not strictly the same thing so I think it's better like that

Reply:
Ok, still requesting changes?